### ISSUE TITLE
Update loc for no-disabled-tests / no-focussed-tests

### ISSUE THREAD

**Issue by Moeriki** [tlvince/eslint-plugin-jasmine#132]
**Do you want to request a *feature* or report a *bug*?**

Improvement.

**What is the current behavior?**

Visually annoying `loc` that covers whole function.

![image](https://user-images.githubusercontent.com/38758/27913359-287f87b6-6260-11e7-9e42-59376405f669.png)

**What is the expected behavior?**

Nice `loc` that covers function name only.

![image](https://user-images.githubusercontent.com/38758/27913410-4a40fdd0-6260-11e7-8bfa-368e01fdc0b8.png)

---

Can make PR.

**Comment by DianaSuvorova** [comment]
👍 Sounds reasonable to me. If you want to make PR - I will review ...

**Comment by Moeriki** [comment]
In the docs for rule authors I noticed the documented usage of `context.report` is passing a single object arg.

```js
context.report({ message, node })
```

This however isn't the case with `no-disabled-tests` and many other tests.

```js
context.report(node, message)
```

Converting the call from the (I presume) older format to the single-object style **fixed my issue**.

Would you mind using this new call-format project-wide?
https://github.com/tlvince/eslint-plugin-jasmine/compare/master...Moeriki:fix-132

Didn't have to touch the tests. All green!

**Comment by DianaSuvorova** [comment]
@Moeriki , Fantastic! 
Why don't you create a PR  (hit "create pull request" here https://github.com/tlvince/eslint-plugin-jasmine/compare/master...Moeriki:fix-132) and I will merge it in.
